subliminal/video.py

```diff
@@ -96,7 +96,7 @@ def fromguess(cls, name, guess):
         """Create an :class:`Episode` or a :class:`Movie` with the given `name` based on the `guess`.
 
         :param str name: name of the video.
-        :param dict guess: guessed data, like a :class:`~guessit.guess.Guess` instance.
+        :param dict guess: guessed data, like a :class:`~rebulk.match.MatchesDict` instance.
```

Comment:
Here is the portion of code to edit: https://github.com/Diaoul/subliminal/blob/master/docs/conf.py#L300-L309